lib/oli_web/live/delivery/student/index_live.ex

```diff
@@ -77,7 +77,7 @@ defmodule OliWeb.Delivery.Student.IndexLive do
                   section
                   |> get_or_compute_full_hierarchy()
                   |> Hierarchy.find_module_ancestor(
-                    page[:resource_id],
+                    Map.get(page, :resource_id),
```

Comment:
NIT: I think both approaches will return `nil` if the key does not exist in the map, or the value if it exists. So I was wondering why we made this change here

Reply:
There is a subtle affect that serializing / deserializing the SectionResource schema has:  `page[:resource_id]` style access of fields no longer works.